cvat-ui/src/components/login-with-token/login-with-token.tsx

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import {Redirect, useParams} from "react-router";
+import {useCookies } from "react-cookie";
+
+export default function LoginWithTokenComponent(){
+    const { sessionId, token } = useParams()
+    const [cookies, setCookie] = useCookies(['sessionid', 'csrftoken'])
+    const expires = new Date(new Date().setFullYear(new Date().getFullYear() + 1))
```

Comment:
Does the cookies actually expire after one year? Does it correspond to the server?

Reply:
token expires after one year. However session id after 2 weeks, thanks for asking. Fixed